src/App.js

```diff
@@ -0,0 +1,51 @@
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import Cookies from 'js-cookie';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import SplashPage from './components/SplashPage';
+import ProtectedRoute from './components/ProtectedRoute';
+import { loginAsync } from './redux/users/userSlice';
+import Doctors from './components/Doctors';
+import Layout from './components/Layout';
+import Details from './components/Details';
+import Appointments from './components/Appointments';
+import NewAppointment from './components/NewAppointment';
+import NewPasswordForm from './authentication/passwords/newPassword';
+import NewDoctor from './Pages/NewDoctor';
+import DeleteDoctor from './components/delete';
+
+const App = () => {
```

Comment:
- [ ] In the mobile implementation, the homepage styling is questionable.  This is the top of the homepage and the first element card is overflowing:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/7f1aeb28-f8d5-404d-9bc8-4e27b27b2878)

- [ ] Also, if I open the navbar, it appears behind the content:

![image](https://github.com/zeff96/doctor_appoitment_front_end/assets/57421823/cf792fd5-7f0e-46f3-a217-e0984f6d010c)

Kindly, touch up your mobile home page so it works and shows as intended. 👍 

_This was tested both in Chrome and Firefox._